frontend/packages/dev-console/integration-tests/support/step-definitions/pipelines/pipelines-triggers.ts

```diff
@@ -0,0 +1,296 @@
+  triggerTemplateDetailsPage.verifyActionsDropdown();
+});
+
+Given('user is at Trigger Template Details page', () => {
+  pipelinesPage.selectPipeline(store.pipeline1);
+  pipelineDetailsPage.verifyPage();
+  pipelineDetailsPage.selectTriggerTemplateLink();
+  triggerTemplateDetailsPage.verifyPage();
+});
+
+When('user clicks on Event listener', () => {
+  triggerTemplateDetailsPage.selectEventListener();
+});
+
+Then('user will be redirected to Event Listener Details page', () => {
+  eventListenerDetailsPage.verifyPage();
+});
+
+Then(
+  'Details tab is displayed with field names Name, Namespace, Labels, Annotations, Created At, Owner, Trigger Templates and Trigger Bindings',
+  () => {
+    eventListenerDetailsPage.verifyFields();
+  },
+);
+
+Given('user is at Event Listener Details page', () => {
+  pipelinesPage.selectPipeline(store.pipeline1);
+  pipelineDetailsPage.verifyPage();
+  pipelineDetailsPage.selectTriggerTemplateLink();
+  triggerTemplateDetailsPage.verifyPage();
+  triggerTemplateDetailsPage.selectEventListener();
+  eventListenerDetailsPage.verifyPage();
+});
+
+When('user clicks on Trigger Binding', () => {
+  eventListenerDetailsPage.selectTriggerBindingLink();
+});
+
+Then('user will be redirected to Cluster Trigger Binding Details page', () => {
+  clusterTriggerBindingDetailsPage.verifyPage();
+});
+
+Then(
+  'Details tab is displayed with field names Name, Labels, Annotations, Created At, Owner',
+  () => {
+    clusterTriggerBindingDetailsPage.verifyFields();
+  },
+);
+
+Then('Actions dropdown display on Cluster Trigger Binding page', () => {
+  clusterTriggerBindingDetailsPage.verifyActionsDropdown();
+  navigateTo(devNavigationMenu.Pipelines);
+});
+
+When('user selects the first option from the Trigger Template drop down field', () => {
+  cy.get(pipelinesPO.removeTrigger.triggerTemplate)
+    .should('be.enabled')
+    .click();
+  cy.get('[data-test-id="dropdown-menu"]').click();
+});
+
+When('user clicks on Remove button', () => {
+  cy.get(pipelinesPO.removeTrigger.remove)
+    .should('be.enabled')
+    .click();
+});
+
+Then('modal is displayed with header message {string}', (headerName: string) => {
+  modal.modalTitleShouldContain(headerName);
+});
+
+Then('trigger template dropdown displayed with help text Select Trigger Template', () => {
+  cy.get(pipelinesPO.removeTrigger.triggerTemplate).should('have.text', 'Select Trigger Template');
+});
+
+Then('Remove button will be disabled', () => {
+  cy.get(pipelinesPO.removeTrigger.remove).should('be.disabled');
+  cy.get(pipelinesPO.removeTrigger.cancel).click();
+});
+
+Then(
+  'option {string} is not available in kebab menu for {string}',
+  (option: string, pipelineName: string) => {
+    pipelinesPage.selectKebabMenu(pipelineName);
+    cy.byTestActionID(option).should('not.be.visible');
+  },
+);
+
+Given('Trigger is added to the pipeline {string} present in pipeline details page', (a: string) => {
+  cy.log(a);
+  // Manual step
```

Comment:
@makambalaji  Is this just a placeholder and will there be a separate PR to add content?

Reply:
This is manual step, To avoid warnings in feature file, I added its step in step definition and commented as manual step @karthikjeeyar